src/uct/tcp/tcp_md.c

```diff
@@ -80,7 +80,12 @@ uct_component_t uct_tcp_component = {
     .rkey_release       = ucs_empty_function_return_success,
     .name               = UCT_TCP_NAME,
     .md_config          = UCT_MD_DEFAULT_CONFIG_INITIALIZER,
-    .cm_config          = UCS_CONFIG_EMPTY_GLOBAL_LIST_ENTRY,
+    .cm_config          = {
+        .name           = "TCP-SOCKCM connection manager",
+        .prefix         = "TCP_",
+        .table          = uct_cm_config_table,
+        .size           = sizeof(uct_cm_config_t),
```

Comment:
no need for trailing `,` at 87 line

Reply:
@dmitrygx we have this comma in all other similar places so i kept here the same:
https://github.com/openucx/ucx/blob/master/src/uct/ib/base/ib_md.c#L1647
https://github.com/openucx/ucx/blob/master/src/uct/sm/mm/base/mm_md.h#L168
https://github.com/openucx/ucx/blob/master/src/uct/sm/knem/knem_md.c#L395
and in all other table configs...

Reply:
np then
;)